style.css

```diff
@@ -0,0 +1,415 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Nunito', sans-serif;
+  scroll-behavior: smooth;
+
+}
+
+.primaryButton {
+  background-color: #f460b7;
+  border: none;
+  border-radius: 8px;
```

Comment:
Why does this button have a different border-radius than the button in the nav.

Reply:
I don't see that this is changed

Reply:
Ive changed it now but in the other button so it matches the buttons in the editor